base/kilda-base-python3/Dockerfile

```diff
@@ -13,10 +13,14 @@
 #   limitations under the License.
 #
 
-FROM ubuntu:xenial
+FROM ubuntu:bionic
 
 ENV DEBIAN_FRONTEND noninteractive
 
+# pick Open VSwitch 2.10 from cosmic release
+RUN echo "deb http://ru.archive.ubuntu.com/ubuntu/ cosmic main restricted" >> /etc/apt/sources.list
```

Comment:
Why it is here (in "base" image) and not in lab-service image?

PS There is a lot of weir packages installed here... for "base" image.

Reply:
Yes, I can make one Docker file with all this stuff for lab-service. And it will rebuild whole image with fetching deb packages and installing them for each docker-compose build. Do you like that?
Or I can rename that image to, for example, 'lab-service-base" if you want.

Reply:
"python3-ubuntu" image renamed to "base-lab-service"